news/models.py

```diff
@@ -7,3 +7,71 @@ class Subscriber(models.Model):
 
     def __str__(self):
         return self.user.username
+
```

Comment:
We also need a Reporter model which should have a foreign key to Subscriber, first_name, last_name, agency, avatar